examples/deadline/All-In-AWS-Infrastructure-SEP/python/README.md

```diff
@@ -0,0 +1,51 @@
+# RFDK Sample Application - Deadline Spot Event Plugin - Python
+
+## Overview
+[Back to overview](../README.md)
+
+## Instructions
+
+---
+**NOTE**
+
+These instructions assume that your working directory is `examples/deadline/All-In-AWS-Infrastructure-SEP/python/` relative to the root of the AWS-RFDK package.
+
+---
+
+1.  This sample app on the `mainline` branch may contain features that have not yet been officially released, and may not be available in the `aws-rfdk` package installed through pip from PyPI. To work from an example of the latest release, please switch to the `release` branch. If you would like to try out unreleased features, you can stay on `mainline` and follow the instructions for building, packing, and installing the `aws-rfdk` from your local repository.
```

Comment:
This still mentions needing to build, pack, and install the local aws-rfdk if working from mainline, which is correct, but the instructions for doing that aren't here now.